checkout-ui-custom/src/partials/Deliver/AddressForm.js

```diff
@@ -172,7 +172,7 @@ const AddressForm = () => {
       required: true,
       type: 'tel',
       helperText: 'We send shipping updates to this number.',
-      minlength: 9,
+      minlength: 12,
```

Comment:
@harlyon are you sure the new requirements are correct?

Please check:
- Up until now customer can enter it like this: 
0721234567 or 721234567 (9 or 10 digits) 
- Customer should not need to enter +27.